### PR TITLE
IGAPP-1339: Move feedback logic

### PR DESCRIPTION
**Pull request by steffenkleinle**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: web/src/components/Layout.tsx [hunk 1]
```diff
@@ -27,6 +21,7 @@ const RichLayout = styled.div`
     &:focus-visible {
       outline: 2px solid ${props => props.theme.colors.textSecondaryColor};
     }
+
```
Comment (sarahsporck) [inline web/src/components/Layout.tsx:24]: Please remove

**Review by sarahsporck — APPROVED** [review]
Nice! Tested in firefox.

File: web/src/components/__tests__/FeedbackToolbarItem.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    fireEvent.click(getByText('feedback:feedback'))
+
+    expect(getByText('FeedbackModal')).toBeTruthy()
+    expect(container.querySelector(`[id="${LAYOUT_ELEMENT_ID}"]`)?.getAttribute('aria-hidden')).toBe('true')
+
+    fireEvent.click(getByText('FeedbackModal'))
+
+    expect(queryByText('FeedbackModal')).toBeFalsy()
+    expect(container.querySelector(`[id="${LAYOUT_ELEMENT_ID}"]`)?.getAttribute('aria-hidden')).toBe('false')
```
Comment (LeandraH) [inline web/src/components/__tests__/FeedbackToolbarItem.spec.tsx:37]: Thanks for adding a test! Maybe we can also test if e.g. the button to send the feedback is visible or not.

**Review by LeandraH — APPROVED** [review]
Nice, tested in Firefox

File: web/src/components/__tests__/FeedbackToolbarItem.spec.tsx (hunk 2 above)
Comment (steffenkleinle) [reply web/src/components/__tests__/FeedbackToolbarItem.spec.tsx:37]: Not part of the toolbar item so won't test it here.

**Pull request merged**